libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtils.kt

```diff
@@ -72,6 +75,11 @@ internal object MapboxRouteLineUtils {
             >,
         List<ExtractedRouteData>> by lazy { LruCache(NUMBER_OF_SUPPORTED_ROUTES) }
 
+    private val extractRouteRestrictionDataCache: LruCache<
+        CacheResultUtils.CacheResultKey1<DirectionsRoute, List<ExtractedRouteRestrictionData>
```

Comment:
Restrictions are not subject to route refresh, so why not use the faster `CacheResultUtils.CacheResultKeyRoute`?

Reply:
I didn't think of it. I'll change it.